packages/adminrouter/extra/src/lib/util.lua

```diff
@@ -243,5 +243,34 @@ function string:strip()
     return self:match "^%s*(.-)%s*$"
 end
 
+local error_response_bodies = {}
+local errorpages_dir_path = os.getenv("AUTH_ERROR_PAGE_DIR_PATH")
+if errorpages_dir_path == nil then
+    ngx.log(ngx.WARN, "AUTH_ERROR_PAGE_DIR_PATH not set.")
+else
+    error_codes = {401, 403, 404, 500, 501, 502, 503, 504}
+    for _, code in ipairs(error_codes) do
+        local p = errorpages_dir_path .. "/" .. code .. ".html"
+        ngx.log(ngx.NOTICE, "Reading " .. code .. " response from `" .. p .. "`.")
```

Comment:
Can you remind me if we currently emit ngx.NOTICE log messages by default?

Reply:
Yes, we long at `INFO` level.